person.rb

```diff
@@ -1,8 +1,13 @@
-class Person
+require_relative 'nameable'
+require_relative 'capitalizedecorator'
+require_relative 'trimmerdecorator'
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
-  def initilize(age, name = 'Unknown', parent_permission: true)
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super
```

Comment:
- [ ] The `ArgumentError` issue which I highlighted previously has not been resolved. Kindly fix this issue to get approved 👍 
![image](https://user-images.githubusercontent.com/46839548/216209617-bad13337-f574-43b6-b0d0-897ccd1a9894.png)

Reply:
Dear @codepantha ,

May you please assist with this one? If I remove _**super**_ it is flagging a linter error, if add it, it is throwing a run error.
May you suggest a way to move around this?